framework/source/class/qx/bom/webfonts/Validator.js

```diff
@@ -33,17 +33,25 @@ qx.Class.define("qx.bom.webfonts.Validator", {
 
   /**
    * @param fontFamily {String} The name of the font to be verified
+   * @param fontWeight {String} the weight of the font to be verified
+   * @param fontStyle {String} the style of the font to be verified
    * @param comparisonString {String?} String to be used to detect whether a font was loaded or not
    * whether the font has loaded properly
    */
-  construct : function(fontFamily, comparisonString)
+  construct : function(fontFamily, fontWeight, fontStyle, comparisonString)
```

Comment:
BC issue here, if people were calling this expecting the comparison string to be the second argument?